backend-server/src/main/java/org/dice/ida/action/process/ActionExecutor.java

```diff
@@ -0,0 +1,36 @@
+package org.dice.ida.action.process;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.dice.ida.action.def.Action;
+import org.dice.ida.constant.IDAConst;
+import org.dice.ida.model.ChatMessageResponse;
+import org.dice.ida.model.Intent;
+
+import com.google.cloud.dialogflow.v2.QueryResult;
+public class ActionExecutor {
+	
+	Action action;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/dice-group/IDA/pullRequest?prid=6037853)